guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc

```diff
@@ -5,7 +5,7 @@ You can customize and rerun the Ansible playbook to reconfigure existing hosts.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Applications > App Instances*.
-. Select the application instance you want to deploy and click the *Run Playbook - customize first* button from the drop down menu.
+. Select the application instance you want to deploy and click *Run Playbook {endash} customize first* from the drop down menu.
```

Comment:
```suggestion
. Select the application instance you want to deploy and select *Run Playbook {endash} customize first* from the drop-down menu.
```
drop-down menu with a hyphen per the IBM guide

"Select" seem like the term to use for drop downs, but I'm not sure to be honest.

Reply:
Applied both suggestions.